doc/src/docs/InputOutputReference/01c-InputOutputReference.md

```diff
@@ -7152,6 +7152,15 @@ An example of this object in an IDF is:
 
 * HVAC,Average,Zone Ideal Loads Hybrid Ventilation Available Status []
 
+* HVAC,Average,Zone Ideal Loads Outdoor Air Mass Flow Rate [kg/s]
+
```

Comment:
@JasonGlazer Please add the variable descriptions further down.  You can adapt the text from the equivalent System Node flow variable descriptions.  Be sure to pull this branch before editing - I've merged in develop and resolved a conflict today.